src/views/SpaceSettings.vue

```diff
@@ -190,6 +194,41 @@ onBeforeRouteLeave(async () => {
 
       <template v-else>
         <div class="mt-3 space-y-3 sm:mt-0">
+          <BaseMessageBlock
```

Comment:
Are we not going with a modal on all pages and different message for admins. Message for admins should have more details/information. Also normal users don't need a link to the docs

Reply:
Removed the docs links for regular user

Reply:
Message for admins with more details is the one in the settings page.

What additional message would you like to show, aside from a link to settings page to reactivate the space ?